src/pin-field/PinField.ts

```diff
@@ -117,22 +117,45 @@ export class PinField extends OmniFormElement {
         }
     }
 
+    _sanitiseValue(value: string) {
+        if (value !== null && value !== undefined && value !== '') {
```

Comment:
Is this not equivalent to ` if(value) ...` ?

Reply:
Updated the if statement accordingly